src/services/game.service.js

```diff
@@ -0,0 +1,51 @@
+const Deck = require('../utils/deck.model');
+
+const isLastRound = (round, playerCount) => {
+  switch (playerCount) {
+    case 3:
+      return round === 20;
+    case 4:
+      return round === 15;
+    case 5:
+      return round === 12;
+    case 6:
+      return round === 10;
+    default:
+      return false;
+  }
+};
+
+exports.startRound = (round, playerCount) => {
+  if (round < 1 || round > 20) { throw new Error('the count of rounds must be from 1 to 20'); }
+
+  if (playerCount < 3 || playerCount > 6) { throw new Error('the player count must be from 3 to 6'); }
+
+  const deck = new Deck();
+  deck.shuffle();
+  const result = {
```

Comment:
We could return this object directly and remove the variable

Reply:
done